.github/workflows/wasm_backend_tests_ci.yml

```diff
@@ -0,0 +1,97 @@
+name: wasm backend CI
+
+on:
+  push:
+    paths:
+      - '!**'
+      - '!**.md'
+      - 'cmd/tools/builders/**.v'
+      - 'vlib/builtin/**.v'
+      - 'vlib/v/ast/**.v'
+      - 'vlib/v/scanner/**.v'
+      - 'vlib/v/scanner/**.v'
```

Comment:
hi, is it right this line (duplicated) ?

Reply:
Fixed in bba3438 .